.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
I'm not aware of any, but trailing whitespace can be significant in Markdown.

(Honestly, I copied the file from another of my own projects, and tweaked the indent settings to match your style)